pages/common/pixi-task.md

```diff
@@ -0,0 +1,20 @@
+# pixi task
+
+> Manage tasks in the project environment.
+> More information: <https://pixi.sh/latest/reference/cli/#task>.
+
+- Create a new task:
+
+`pixi task add {{task-name}} {{task-command}}`
```

Comment:
```suggestion
`pixi task add {{task_name}} {{task_command}}`
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides%2Fstyle-guide.md#naming

![IMG_20240710_115037.jpg](https://github.com/tldr-pages/tldr/assets/155078792/d85aba9e-7635-403b-b729-c580ef4a3925)

Reply:
I was utterly blind. I read CONTRIBUTING.MD but for some reason did not see the style guide linked in it. Sorry for having you fix these.

Reply:
Went through the style guide and the files, everything should be according to the style guide now